xbmc/cores/VideoPlayer/DVDCodecs/Video/DXVA.cpp

```diff
@@ -342,6 +330,15 @@ bool CContext::CreateContext()
     return false;
   }
 
+  if (FAILED(hr) || !m_sharingAllowed)
+  {
+    // enable multi-threaded protection only if is used same d3d11 device for rendering and decoding
+    ComPtr<ID3D11Multithread> multithread;
+    hr = pD3DDevice.As(&multithread);
+    if (SUCCEEDED(hr))
+      multithread->SetMultithreadProtected(1);
```

Comment:
if you'r enabling multi-thread protection here you have to disable it after context is completely destroyed. in other case you enable it for the whole application on first playback even if no more playback is used.

Reply:
> you enable it for the whole application on first playback even if no more playback is used.

Correct: in systems do not support discrete decoder (if any) is enabled whole application all the time **same as before** for all systems with/without discrete decoder.